interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: Good question. A parameter is a placeholder for the value, it isn't the value itself as it is the middle man in a way. An argument is the value that we want to invoke. 
 
-Researched answer:
+Researched answer: A parameter is the function name's definitions or placeholders. A parameter is the category while an argument is the value. For example a parameter is a,b and when we execute the code during the console log we want to invoke a,b by using using numbers 5,10 (we are adding these two numbers in this instance) those are the arguments or the values to the parameters and when the console log is ran we would return 15. 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-Your answer:
+Your answer: Great question. The value, index, array are the three predefined paramaters for the .map() method. The value would be one of the six primitive data types (boolean, number, string, null, undefined, and or symbol). The value is a mandatory part of the array while the index and array are not. The index could be a specified number ([1]) indiciating what index of the value were trying to access or it could simply be a number (7). 
 
-Researched answer:
+Researched answer: The value, index, and array are the predefined parameters and the only required parameter is the value. The array is the array of the current element (value). 
 
 3. What is the difference between map and filter?
 
-Your answer:
+Your answer: Good question. The .map() method is iterating through an object, array, or string and returning a new array with the exact same length. The .filter() method iterates through an object, array, or string until the condition is satisified or the condition is met. 
```

Comment:
Just a note, .map and .filter are array methods.  They won't work on object or string. :)